### PR TITLE
WIP: Updated webpack config to include translations from spring app

### PR DESCRIPTION
**Pull request by ericenns**
## Description of changes
Webpack will now create separate `en` and `fr` bundles so that translations from spring app can be used in js.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [ ] Tests added (or description of how to test) for any new features.
* [ ] User documentation updated for UI or technical changes.

### CODE REVIEW

**Comment by ericenns** [comment]
WIP as thymeleaf imports have not yet been updated to request js bundles with lang in them

before:
`app.bundle.js`

after:
`app.en.bundle.js`
`app.fr.bundle.js`

Caveat: All bundles even vendor are duplicated now, might be something that we can do in webpack to fix this.

**Comment by ericenns** [comment]
All pages but bio_hansel results have been updated, as I can't seem to find the translations for it.

File: src/main/webapp/pages/projects/project_linelist.html [hunk 1]
```diff
@@ -11,162 +11,6 @@
...
-                  empty: /*[[#{linelist.editing.undo.empty}]]*/ "The samples metadata was removed"
-                }
-              },
-              selected: {
-                none: /*[[#{linelist.selected.none}]]*/ "No samples selected",
-                one: /*[[#{linelist.selected.one}]]*/ "One sample selected",
-                multiple: /*[[#{linelist.selected.multiple}]]*/ "Many samples selected"
-              },
-              error: {
-                message: /*[[#{linelist.error.message(${project.getLabel()})}]]*/ "DANGER WILL ROBINSON"
```
Comment (joshsadam) [inline src/main/webapp/pages/projects/project_linelist.html]: This is going to be an issue.  Here the message requires the project name passed into it.  He could do that in the javascript code pretty easily, but we need to make sure that it is done. e.g.:
```javascript
__("linelist.error.message").replace("{0}, project.label); // or something like that.
```

Comment (ericenns) [reply src/main/webapp/pages/projects/project_linelist.html]: I fixed this, project.getLabel returns the name and the project hash in window.PAGE has name attribute to I accessed it that way.

**Comment by ericenns** [comment]
Closed in favour of incoming PR

**Pull request closed without merging**